packages/react-dom/src/__tests__/ReactFunctionComponent-test.js

```diff
@@ -360,36 +411,40 @@ describe('ReactFunctionComponent', () => {
     );
   });
 
-  it('should provide a null ref', () => {
```

Comment:
This was testing ReactTestUtils not function components. I opted to instead adjust the `should warn when given a function ref` which had an assertion that we never hit.